pkg/cloud_provider/file/file.go

```diff
@@ -31,7 +32,7 @@ import (
 	"google.golang.org/api/option"
 	"google.golang.org/grpc/codes"
 	"k8s.io/apimachinery/pkg/util/wait"
-	"k8s.io/klog"
+	"k8s.io/klog/v2"
```

Comment:
hmm I wonder if moving to v2 will affect other logging as well. V1 is deprecated so we SHOULD be using v2 but is v2 backwards compatible? @msau42 do you know?

Reply:
The Verbose type changed which caused klog to rev to v2: https://github.com/kubernetes/klog/commit/cd60aa438f9c750641d5feacd2999e58df8af28f

This may also be important if we are supporting both klog v1 AND v2 AND glog? We probably want to convert it all https://github.com/kubernetes/klog#coexisting-with-klogv2